charts/prefect-server/templates/serviceaccount.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
```

Comment:
should we create a separate service account per deployment? we don't grant any permissions to these service accounts?

Reply:
hrmmm. i feel like it'd be OK to keep them combined - but realistically, chart authors would probably future proof this by creating a separate SA or making each SA name override-able